app/models/chart.rb

```diff
@@ -1,15 +1,8 @@
 class Chart < ActiveRecord::Base
   has_many :data
-  accepts_nested_attributes_for :data, reject_if: :all_blank, allow_destroy: true
```

Comment:
there is individual validation in datum.rb going on.
if we set `reject_if` and `allow_destroy` our error-messages become less descriptive.